app/views/inventory_foods/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+  <div class="row d-flex justify-content-center">
+    <div class="mt-5 p-2 border shadow-lg bg-white rounded">
+      <h4 class="text-center"><%= @inventory.name %></h4>
+      <% unless @foods.empty? %>
+        <%= render 'new' %>
+      <% else %>
+        <p class="text-center mt-4">
+          No foods available. Please add a food first. <br>
+          <%= link_to 'Add a Food', new_food_path, data: { turbo_frame: "remote_modal" } %>
+        </p>
+        <div class="actions text-end mt-4">
+          <%= link_to 'Cancel', inventory_path(@inventory), class: "btn btn-dark btn-sm px-3" %>
+        </div>
+      <% end %>
+    </div>
+  </div>
+</div>
```

Comment:
![image](https://github.com/GhausAliKhan/recipe-app/assets/93039359/863ac74b-c147-40df-9a2e-24f1b131e86d)

- [ ] It would be nice if you fix this error which comes while deleting an inventory food :fearful: Kindly remove this error to make your app error free :+1: .